api/python/quilt3/packages.py

```diff
@@ -748,13 +748,21 @@ def set_dir(self, lkey, path=None, meta=None):
             path(string): path to scan for files to add to package.
                 If None, lkey will be substituted in as the path.
             meta(dict): user level metadata dict to attach to lkey directory entry.
+            update_policy(str): can be either 'incoming' (default) or 'existing'.
+                If 'incoming', whenever logical keys match, always take the new entry from set_dir.
+                If 'existing', whenever logical keys match, retain existing entries
+                and ignore new entries from set_dir.
 
         Returns:
             self
 
         Raises:
             When `path` doesn't exist
+            ValueError: if the 'update_policy' is not a PackageUpdatePolicy enum value.
         """
+        if update_policy not in PACKAGE_UPDATE_POLICY:
+            raise ValueError(f"{update_policy} is not a valid update policy")
```

Comment:
We should tell users how to fix the problem, not just "there was a problem" :)
```suggestion
            raise ValueError(f"Update policy should be one of {str(PACKAGE_UPDATE_POLICY)}, not {update_policy}")
```